testfixtures.go

```diff
@@ -89,21 +85,33 @@ func newContext(db *sql.DB, helper Helper, fixtures []*fixtureFile) (*Context, e
 	return c, nil
 }
 
+// CheckDatabaseName returns nil if databaseName matches regexp
+//     if err := fixtures.CheckDatabaseName(); err != nil {
+//         log.Fatal(err)
+//     }
+func (c *Context) CheckDatabaseName() error {
```

Comment:
I'm thinking of renaming this to something like `DetectTestDatabase`. What do you think?

Also, I think we should return `(bool, error)` instead of having a custom error type

Reply:
Returning (bool, error) will cause the checking of both bool result and error. And fail result of bool checking will lead to creating new error message. Seems it is not a good design